packages/backend/src/api/routes/profile.ts

```diff
@@ -14,9 +15,9 @@ export const initProfileRoutes = (
 
   router.get(
     apiPath(path, ProfileApiRoutes.GET_USER_BY_ID),
-    wrap((req: Request) => {
+    wrap(async (req: Request) => {
       const { userId } = req.params;
-      const user = profileService.getUser({ userId });
+      const user = profileService.getUser({ userId, req });
```

Comment:
Why req is in service

Reply:
I didn't create a new approach. I just used what i saw in anothe routes, which was created before my task. I passed this parametr for translation of errors

Reply:
req.t('some text of error')